### DISCUSSION TITLE
文件无法预览

### DISCUSSION THREAD

**Discussion by realzolo** [spencerwooo/onedrive-vercel-index#440] [Q&A] [answered]
地址: store.onezol.com
我只改了配置文件/config/site.config.js
![image](https://user-images.githubusercontent.com/63219216/153891560-824a2867-654c-46ad-b628-124099b770d5.png)
![image](https://user-images.githubusercontent.com/63219216/153892240-39909fcf-1d19-49a0-a597-ca386774da4e.png)
![image](https://user-images.githubusercontent.com/63219216/153891768-1f2b3a13-ce6f-4cbe-8311-a3b307a2c22b.png)

所有文件都无法预览  也下载不了

**Comment by spencerwooo** [comment, marked as answer]
是 OneDrive 国际版吗？我在国际版上成功复现了。

**Reply by realzolo** [reply]
对的,国际版

**Reply by spencerwooo** [reply]
should be fixed in 049be5d841ce6279427dbcdda8902e1606fac55e

**Reply by realzolo** [reply]
thanks.
